shared/wallets/asset/index.js

```diff
@@ -90,58 +84,81 @@ type BalanceSummaryProps = {
   total: string,
 }
 
-const BalanceSummary = (props: BalanceSummaryProps) => (
-  <Box2 direction="vertical" fullWidth={true} style={styles.balanceSummaryContainer}>
-    <Divider style={{marginBottom: globalMargins.tiny}} />
-    <Box2 direction="horizontal" fullWidth={true}>
-      <Text type="BodySemibold" style={styles.leftColText}>
-        Total
-      </Text>
-      <Text type="BodyExtrabold" selectable={true}>
-        {props.total}
-      </Text>
-    </Box2>
-    {props.reserves.map(reserve => (
-      <Box2 direction="horizontal" fullWidth={true} key={reserve.description}>
-        <Text type="Body" lineClamp={1} style={styles.leftColText}>
-          Reserve ({reserve.description})
-        </Text>
-        <Text type="Body" lineClamp={1} selectable={true}>
-          -{reserve.amount}
-        </Text>
-      </Box2>
-    ))}
-    <Divider style={{marginBottom: globalMargins.tiny, marginTop: globalMargins.tiny}} />
-    <Box2 direction="horizontal" fullWidth={true} style={{alignItems: 'flex-start'}}>
-      <Text type="BodySemibold" style={styles.leftColText}>
-        Available to send
-      </Text>
-      <Box2 direction="vertical" style={styles.balanceContainer}>
-        <Text type="Body" selectable={true} style={{fontWeight: '800'}}>
-          {props.availableToSend}
-        </Text>
-        <Text type="BodySmall" selectable={true}>
-          {props.equivAvailableToSend}
-        </Text>
-      </Box2>
-    </Box2>
-  </Box2>
-)
+const BalanceSummary = (props: BalanceSummaryProps) => {
+  const _openStellarURL = () => {
```

Comment:
can we make this a prop thats made in the contianer instead. that can send null or this if its mobile and this 'dumb' component can just pass this.props.openStellarURL to the onClick

Reply:
Yup!